### ISSUE TITLE
dockercfg.php fails on require('password.php')

### ISSUE THREAD

**Issue by gabeos** [Cydrobolt/docker-polr#1]
Seems like password.php got renamed to lib-password.php, while the script wasn't updated accordingly. 
minor thing, just a minor inconvenience to rebuild the image since the php script gets deleted on start-up after failing so you can't run it manually. anyway, just an fyi :)

thanks for polr!

**Comment by gabeos** [comment]
Also, I had some issues with the SETUP_PASSWORD settings:
 When left alone, <url>/setup.php would not allow me to access the setup page, despite having the correct password. As a workaround, I removed the setting from being written to config in dockercfg.php. 

Also, the /app directory needs to be write-able to www-data in the case that SETUP_PASSWORD is specified, since it has to write the config.php file to that directory. This caused an error on submitting the setup.php page.

**Comment by cydrobolt** [comment]
Yup, you're right. Didn't notice that the filename wasn't updated. I don't maintain the Docker image, although I could try to fix it.

Sorry for such a late reply, I wasn't watching the repo.

**Comment by gabeos** [comment]
Thanks, it's not a big deal. I got it working for myself, just wanted to document what i found. I saw you changed the require dep to lib-password.php :+1: 

Other than that, I just removed 

```
if [ $SETUP_PASSWORD == "none" ]; then
    rm setup.php;
fi
```

from run.sh
and removed `$ppass = ...` from dockercfg.php when setting $data. However, then there's no way to set a setup password. Your choice on keeping it. Possibly changing the permissions I mentioned would help, not sure though.
